CMakeLists.txt

```diff
@@ -258,3 +267,7 @@ endif()
 
 # Add packaging
 add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/cmake/packaging)
+
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.18 AND NOT DEFINED EMSCRIPTEN)
```

Comment:
I would make CMake 3.18 our minimum required version.

Reply:
Currently we can't as the Ubuntu 18.04 has CMake version 3.10 on it.  I'm not sure what version Ubuntu 20.04 has.

Reply:
This is used for the utility target of generating the hex form of the MathML DTD, so not something that anyone should have to use.

Reply:
> Currently we can't as the Ubuntu 18.04 has CMake version 3.10 on it. I'm not sure what version Ubuntu 20.04 has.

FWIW, Ubuntu 20.04 currently uses CMake 3.16.3.

> This is used for the utility target of generating the hex form of the MathML DTD, so not something that anyone should have to use.

Fair enough, but then shouldn't there be an option for it?